src/ReactiveUI/Interfaces/IPropertyBindingHook.cs

```diff
@@ -17,13 +17,11 @@ public interface IPropertyBindingHook
         /// Called when any binding is set up.
         /// </summary>
         /// <returns>If false, the binding is cancelled.</returns>
-        /// <param name="source">The source ViewModel.</param>
         /// <param name="target">The target View (not the actual control).</param>
         /// <param name="getCurrentViewModelProperties">Get current view model properties.</param>
         /// <param name="getCurrentViewProperties">Get current view properties.</param>
         /// <param name="direction">The Binding direction.</param>
         bool ExecuteHook(
-            object source,
```

Comment:
This is the part that concerns me about this PR with a breaking API change (not related to nullability)